cpp/src/ray/config_internal.cc

```diff
@@ -48,6 +48,14 @@ ABSL_FLAG(std::string, ray_logs_dir, "", "Logs dir for workers.");
 
 ABSL_FLAG(std::string, ray_node_ip_address, "", "The ip address for this node.");
 
+ABSL_FLAG(std::string, ray_include_dashboard, "",
```

Comment:
Define bool is better？

Reply:
I don't use bool because I can not know if this flag has been set by bool type. Flags has a higher priority than the configuration. It will lead to invalid config.